### DISCUSSION TITLE
Light mode support on documentation

### DISCUSSION THREAD

**Discussion by heybran** [timber/timber#3014] [Ideas]
Hi,

My eye doesn't feel comfortable reading dark mode documentation, if I or someone else wants to contributing by adding light mode support to Timber's documentation site, how should we get started on that?

I bet some other people also have a hard time reading dark mode, so I would like to make this happen.

**Comment by gchtr** [comment]
Hey @heybran

We have an open issue for that over on the documentation repository on https://github.com/timber/docs/issues/78. We agree that it would make sense to have a light mode for the documentation. But we had to focus on other more urgent things in the past couple months.

If you would like to contribute, it’s probably best to start there. The dark styles were added in https://github.com/timber/docs/pull/23, so it shouldn’t be too hard to restore the light styles from the changes there and apply them as separate styles with a dark mode switcher.

If you want to install the docs, you should find the relevant information in the [README of the repository](https://github.com/timber/docs).

**Reply by heybran** [reply]
Thanks @gchtr 

I will check the documentation repository.
